back/engines/commercial/insights/spec/acceptance/tasks/category_suggestions_spec.rb

```diff
@@ -29,8 +29,18 @@
   let(:view_id) { view.id }
 
   get 'web_api/v1/insights/views/:view_id/tasks/category_suggestions' do
-    parameter :inputs, 'An array of input identifiers'
-    parameter :categories, 'An array of category identifiers'
+    route_description <<~DESC
+      Get the list of ongoing category-suggestion tasks for a given list
+      of categories or inputs. The list of inputs is not provided explicitly
+      (list of identifiers), but via input filters.
+    DESC
+
+    parameter :categories, 'An array of category identifiers', required: false
+    with_options scope: :inputs, required: false do
+      parameter :categories, 'Filter inputs by categories (union)'
+      parameter :keywords, 'Filter inputs by keywords (identifiers of keyword nodes)'
```

Comment:
Those three parameters are arrays of strings. The types can be explicitly documented with something like:
```ruby
parameter :two_level_array, type: :array, items: {type: :array, items: {type: :string}}
```
(very similar -- probably a mapping -- to an OpenAPI specs). But this is optional and it's not something we do today, just because `rspec-api-documentation` is supppose to infer those types from the actual examples. I think it does it, but I just checked to documentation, and true that is doesn't appear clearly. Let's just add the type explicitly here and see if it produces a better doc.

Personally, I'd like to switch to an OpenAPI specification for our documentation. It's one of the possible output formats of rspec-api-documentation, but this will surely require some rework. It might be a nice 10% project.

Reply:
It doesn't seem to change much:
https://developers.citizenlab.co/api-docs/ee/frontweb_api/IN-652/create-classification-tasks-from-input-filter/category-suggestion_tasks/creates_tasks_only_for_the_relevant_inputs.html